img2txt.py

```diff
@@ -22,7 +22,7 @@ def main(opt):
     if opt.mode == "simple":
         CHAR_LIST = '@%#*+=-:. '
     else:
-        CHAR_LIST = "$@B%8&WM#*oahkbdpqwmZO0QLCJUYXzcvunxrjft/\|()1{}[]?-_+~<>i!lI;:,\"^`'. "
+        CHAR_LIST = r"$@B%8&WM#*oahkbdpqwmZO0QLCJUYXzcvunxrjft/\|()1{}[]?-_+~<>i!lI;:,\"^`'. "
```

Comment:
I think this is not correct, considering
```py
>>> len(r"\"")
2
>>> r"\""[0]
'\\'
>>> r"\""[1]
'"'
```
So there will be two `\` in `CHAR_LIST` (previously there is zero).

My suggestion is
```suggestion
        CHAR_LIST = r"""$@B%8&WM#*oahkbdpqwmZO0QLCJUYXzcvunxrjft/\|()1{}[]?-_+~<>i!lI;:,"^`'. """
```

Reply:
Oh,thanks for your advice